RC_RX_CABELL_V3_FHSS.ino

```diff
@@ -38,9 +38,16 @@
 #include "RX.h"
 #include "Pins.h"
 
+//--------------------------------------------------------------------------------------------------------------------------
+// which sketch is running?
+void showID(void) {
```

Comment:
showID() prints the name/build date of the sketch on startup.  I find this helpful when I try to figure out what code a controller is running after a few weeks.